tests/commands/run_test.py

```diff
@@ -466,6 +466,12 @@ def test_all_push_options_ok(cap_out, store, repo_with_passing_hook):
     assert b'Specify both --origin and --source.' not in printed
 
 
+def test_checkout_type(cap_out, store, repo_with_passing_hook):
+    args = run_opts(origin='', source='', checkout_type='1')
+    ret, printed = _do_run(cap_out, store, repo_with_passing_hook, args)
+    assert ret == 0
```

Comment:
this doesn't really assert anything?

Reply:
This test is here to provide coverage of this line: https://github.com/pre-commit/pre-commit/pull/1339/files#diff-d58a98662b1c7c5efc1562af39480143R320. Without this test, coverage goes down to 99%.

Reply:
the test should probably demonstrate something about that then (perhaps print out in the hook and show that it was passed through)